airflow/utils/dag_processing.py

```diff
@@ -834,7 +834,7 @@ def _log_file_processing_stats(self, known_file_paths):
             rows.append((file_path, processor_pid, runtime, num_dags, num_errors, last_runtime, last_run))
 
         # Sort by longest last runtime. (Can't sort None values in python3)
```

Comment:
This seems to suggest that it's sorting the results by the length of each DAG's last runtime. Is your change sorting by the time of the last runtime? Or was it not sorting in the way the comment suggests and you're just fixing it?

If it's the former, can you update the comment to reflect the new sort order?

Reply:
The comment and the code was not consistent, as you can see from the code above
```
rows.append((file_path, processor_pid, runtime, num_dags, num_errors, last_runtime, last_run))
```
index (5) is last_runtime. I'm just making the code and comment consistent.

It was probably that when more fields were added into the row, and sort field was not updated.